qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|https?:\/\/.+"/>
```

Comment:
@JimDeanSpivey The actual use cases where we use urls is in e.g.:
```
/**
 * Blah blah, from http://long.url.here
 */
```
or
/**
 * Blah blag
 * @see http://long.url.here
 */
```
So my regexp is not correct, should be more like: `^ +\* .*https?:\/\/.+`